hedera-node/src/main/java/com/hedera/services/tokens/HederaTokenStore.java

```diff
@@ -259,6 +231,12 @@ public ResponseCodeEnum wipe(AccountID aId, TokenID tId, boolean skipKeyCheck) {
 			}
 			var account = ledger.getTokenRef(aId);
 			var balance = account.getTokenBalance(tId);
+
+			if(wipingAmount < 0) {
+				return INVALID_WIPING_AMOUNT;
+			}
+			balance -= wipingAmount;
+
 			var outcome = account.wipeTokenRelationship(tId);
```

Comment:
Should this be called?

Reply:
That's a good question. 
1) If the wiping amount is not setting the balance to `0` the reference should be there
2) If the wiping amount is setting the balance to `0`: We need to sync it with the team because:
- If there is a KYC or Freeze flag set it will remove it which may be an effect that is not desired.